### PR TITLE
Create license.md

### PR DESCRIPTION
**Pull request by saraford**
Adding a license.md file to the repo to match the MIT license found in the package.json file https://github.com/electron/electron-api-demos/blob/master/package.json#L35

### CODE REVIEW

File: license.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (MarshallOfSound) [inline license.md]: Shouldn't this be Github Inc.

https://github.com/electron/electron/blob/master/LICENSE#L1

**Comment by saraford** [comment]
@MarshallOfSound good catch! I've updated the copyright to match what's used in the electron repo https://github.com/electron/electron/blob/master/LICENSE

**Comment by jlord** [comment]
Thanks y'all! I updated the year to 2016 and merged the branch ✨

**Comment by MarshallOfSound** [comment]
@jlord I'm not sure what year needs to be in license files but do we have to update other repo's to be the latest year?

Never really though about that 😆

**Comment by jlord** [comment]
I did this year on this repository because I didn't feel we could license it before it existed 😄 I don't think you need to update other licenses. 

I just checked [choosealicense.com](http://choosealicense.com/licenses/mit/) and it says this, which makes me feel like we don't need to update and that this one at 2016 is correct:

> Replace [year] with the current year and [fullname] with the name (or names) of the copyright holders.

**Pull request merged**